<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>question_of_player_To_pvp_mode</class>
 <widget class="QDialog" name="question_of_player_To_pvp_mode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>430</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Segoe UI&quot;;
font: 700 italic 18pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Question_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 12pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>The First options</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Question_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The second options</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Question_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The Third options</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Question_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>The Fourth options</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 12pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Question</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Click To see Qusetion and options</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>Question_1</tabstop>
  <tabstop>Question_2</tabstop>
  <tabstop>Question_3</tabstop>
  <tabstop>Question_4</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>question_of_player_To_pvp_mode</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>destroyed()</signal>
   <receiver>question_of_player_To_pvp_mode</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
